scripts/github-trending.js

```diff
@@ -0,0 +1,68 @@
+// Description:
+//   get the latest trending repositories on github
+//
+// Dependencies:
+//   cheerio
+//   request
+//
+// Configuration:
+//   None
+//
+// Commands:
+//   trending          : list trending repositories (all languages)
+//   trending-language : list trending repositories for the given language
+//   trending project  : get general information for the given project
+//
+
+var cheerio = require('cheerio');
```

Comment:
@SpoonBoy You'll also need to add cheerio in `packages.json`: `npm install --save cheerio`.